src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/ServiceDescriptionService.java

```diff
@@ -534,6 +581,71 @@ public ServiceDescriptionType refreshServiceDescription(Long id, boolean ignoreW
         throw new NotImplementedException("REST ServiceDescription refresh not implemented yet");
     }
 
+    /**
+     * Refresh REST service description
+     *
+     * @param serviceDescriptionType
+     * @return {@link ServiceDescriptionType}
+     * @throws WrongServiceDescriptionTypeException if service type is not REST
+     * @throws ServiceNotFoundException             if there is no service linked to service description
+     * @throws InvalidUrlException                  if url is invalid
+     */
+    private ServiceDescriptionType refreshRESTServiceDescription(ServiceDescriptionType serviceDescriptionType)
```

Comment:
Why does this method / functionality actually exist? Doesn't it only update refreshedDate?

Reply:
This is what I though too, but this was clearly stated in the task description.

'when REST API URL type is "REST API base path" => "REST":
validating that the URL is syntactically correct'

Reply:
Also this is what old ui does.